.github/workflows/copyright-validate.yaml

```diff
@@ -0,0 +1,19 @@
+name: Check Copyright Headers
+
+on:
+  # - pull_request [should we include for all types]
+  push:
```

Comment:
Right now, this is targeted at this specific branch push. But the idea is to run this job for every pull request